build.gradle

```diff
@@ -0,0 +1,104 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+    id 'jacoco'
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    // JPF-core jars (adjust path as needed)
+    implementation files("/home/finn/jpf-push/jpf-core/build/jpf.jar")
```

Comment:
Please make sure that paths are relative, as automated builds cannot "adjust the path as needed".